src/components/TransactionDetails/index.js

```diff
@@ -495,12 +495,12 @@ export default function TransactionDetails(props) {
     )
   }
 
-  const setFromCustom = () => {
+  const setFromCustom = useCallback(() => {
     setActiveIndex(4)
     inputRef.current.focus()
     // if there's a value, evaluate the bounds
     checkBounds(debouncedInput)
-  }
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
so this is where i think we can't use `eslint-disable-line react-hooks/exhaustive-deps`. we're calling setFromCustom whenever someone clicks the custom select, and inside of that i think we'll have a stale version of `debouncedInput`. does this need to use `useCallback` at all?

Reply:
Yep don't even need a callback here

Reply:
yeah it seems like this doesn't need to use a callabck at all - the reason I have it there is because of another warning here 
<img width="328" alt="Screen Shot 2019-09-05 at 1 55 31 PM" src="https://user-images.githubusercontent.com/17100331/64366749-e1d77e00-cfe4-11e9-958a-40fb3b663b52.png">

resulting from this

<img width="701" alt="Screen Shot 2019-09-05 at 1 55 24 PM" src="https://user-images.githubusercontent.com/17100331/64366797-ff0c4c80-cfe4-11e9-84b3-65ae124b6d72.png">

Reply:
In newest commit added the rigth dependencies - were good now